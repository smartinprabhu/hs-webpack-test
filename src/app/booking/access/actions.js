import { CALL_API } from '../../../middleware/api';

export const SAVE_ACCESS = 'SAVE_ACCESS_DATA';
export const SAVE_ACCESS_SUCCESS = 'SAVE_ACCESS_DATA_SUCCESS';
export const SAVE_ACCESS_FALURE = 'SAVE_ACCESS_DATA_FALURE';

export const GET_BOOKING_LIST = 'GET_BOOKING_LIST';
export const GET_BOOKING_LIST_SUCCESS = 'GET_BOOKING_LIST_SUCCESS';
export const GET_BOOKING_LIST_FAILURE = 'GET_BOOKING_LIST_FAILURE';

export function saveAccess(siteId, shiftId, employeeId) {
  return {
    [CALL_API]: {
      endpoint: `access/check?site_id=${siteId}&shift_id=${shiftId}&employee_id=${employeeId}`,
      types: [SAVE_ACCESS, SAVE_ACCESS_SUCCESS, SAVE_ACCESS_FALURE],
      method: 'PUT',
    },
  };
}

export function getBookingList(uuid) {
  return {
    [CALL_API]: {
      endpoint: `booking/uuid/show?uuid=${uuid}`,
      types: [
        GET_BOOKING_LIST,
        GET_BOOKING_LIST_SUCCESS,
        GET_BOOKING_LIST_FAILURE,
      ],
      method: 'GET',
    },
  };
}
