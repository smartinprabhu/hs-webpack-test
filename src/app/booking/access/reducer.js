import {
  SAVE_ACCESS, SAVE_ACCESS_SUCCESS, SAVE_ACCESS_FALURE,
  GET_BOOKING_LIST, GET_BOOKING_LIST_SUCCESS, GET_BOOKING_LIST_FAILURE,
} from './actions';

const initialState = {
  accessInfo: {},
  bookingListInfo: {},
};

// eslint-disable-next-line default-param-last
function accessReducer(state = initialState, action) {
  // eslint-disable-next-line no-param-reassign
  state.accessInfo = { err: null };
  switch (action.type) {
    case SAVE_ACCESS:
      return {
        ...state,
        accessInfo: (state.accessInfo, { loading: true }),
      };
    case SAVE_ACCESS_SUCCESS:
      return {
        ...state,
        accessInfo: (state.accessInfo, { loading: false, data: action.payload.data, err: {} }),
      };
    case SAVE_ACCESS_FALURE:
      return {
        ...state,
        accessInfo: (state.accessInfo, { loading: false, err: action.error.data, data: {} }),
      };
    case GET_BOOKING_LIST:
      return {
        ...state,
        bookingListInfo: (state.bookingListInfo, {
          loading: true, err: {},
        }),
      };
    case GET_BOOKING_LIST_SUCCESS: return {
      ...state,
      bookingListInfo: (state.bookingListInfo, {
        loading: false, data: action.payload.data, err: {},
      }),
    };
    case GET_BOOKING_LIST_FAILURE: return {
      ...state,
      bookingListInfo: (state.bookingListInfo, {
        loading: false, err: action.error.data, data: {},
      }),
    };
    default:
      return state;
  }
}

export default accessReducer;
