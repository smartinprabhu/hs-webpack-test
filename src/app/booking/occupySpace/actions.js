import { CALL_API } from '../../../middleware/api';

export const OCCUPY_SPACE = 'OCCUPY_SPACE';
export const OCCUPY_SPACE_SUCCESS = 'OCCUPY_SPACE_SUCCESS';
export const OCCUPY_SPACE_FALURE = 'OCCUPY_SPACE_DATA';

export function occupySpace(bookingId) {
  return {
    [CALL_API]: {
      endpoint: `space/occupy?emply_shift_id=${bookingId}`,
      types: [OCCUPY_SPACE, OCCUPY_SPACE_SUCCESS, OCCUPY_SPACE_FALURE],
      method: 'POST',
    },
  };
}
