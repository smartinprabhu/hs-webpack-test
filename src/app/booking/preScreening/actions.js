import { CALL_API } from '../../../middleware/api';
import AuthService from '../../util/authService';

export const PRE_SCREENING_PROCESS = 'PRE_SCREENING_PROCESS';
export const PRE_SCREENING_PROCESS_SUCCESS = 'PRE_SCREENING_PROCESS_SUCCESS';
export const PRE_SCREENING_PROCESS_ERROR = 'PRE_SCREENING_PROCESS_ERROR';

export const GET_QUESTIONARY_DATA = 'GET_QUESTIONARY_DATA';
export const GET_QUESTIONARY_DATA_SUCCESS = 'GET_QUESTIONARY_DATA_SUCCESS';
export const GET_QUESTIONARY_DATA_ERROR = 'GET_QUESTIONARY_DATA_ERROR';

export function questionaireDataGetCall(checkListId) {
  return {
    [CALL_API]: {
      endpoint: `prescreen/questionnaire/?ids=[${checkListId}]`,
      types: [GET_QUESTIONARY_DATA, GET_QUESTIONARY_DATA_SUCCESS, GET_QUESTIONARY_DATA_ERROR],
      method: 'GET',
      payload: AuthService.getAccessToken,
    },
  };
}

export function preScreeningPostCall(data) {
  return {
    [CALL_API]: {
      endpoint: 'prescreen/questionnaire/response/create',
      types: [PRE_SCREENING_PROCESS, PRE_SCREENING_PROCESS_SUCCESS, PRE_SCREENING_PROCESS_ERROR],
      method: 'POST',
      payload: { values: data },
    },
  };
}
