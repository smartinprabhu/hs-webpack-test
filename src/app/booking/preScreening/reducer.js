import {
  PRE_SCREENING_PROCESS, PRE_SCREENING_PROCESS_SUCCESS, PRE_SCREENING_PROCESS_ERROR,
  GET_QUESTIONARY_DATA, GET_QUESTIONARY_DATA_SUCCESS, GET_QUESTIONARY_DATA_ERROR,
} from './actions';

const initialState = {
  questionarieData: {},
  preScreeningProcess: {},
};

// eslint-disable-next-line default-param-last
function preScreeningProcessReducer(state = initialState, action) {
  // eslint-disable-next-line no-param-reassign
  state.preScreeningProcess = {};
  switch (action.type) {
    case PRE_SCREENING_PROCESS:
      return {
        ...state,
        preScreeningProcess: (state.preScreeningProcess, { loading: true }),
      };
    case PRE_SCREENING_PROCESS_SUCCESS:
      return {
        ...state,
        preScreeningProcess: (state.preScreeningProcess,
        { loading: false, data: action.payload.data, err: {} }),
      };
    case PRE_SCREENING_PROCESS_ERROR:
      return {
        ...state,
        preScreeningProcess: (state.preScreeningProcess,
        { loading: false, err: action.error.data, data: {} }),
      };
    case GET_QUESTIONARY_DATA:
      return {
        ...state,
        questionarieData: (state.questionarieData, { loading: true }),
      };
    case GET_QUESTIONARY_DATA_SUCCESS:
      return {
        ...state,
        questionarieData: (state.questionarieData,
        { loading: false, data: action.payload.data, err: {} }),
      };
    case GET_QUESTIONARY_DATA_ERROR:
      return {
        ...state,
        questionarieData: (state.questionarieData,
        { loading: false, err: action.payload, data: {} }),
      };
    default:
      return state;
  }
}

export default preScreeningProcessReducer;
