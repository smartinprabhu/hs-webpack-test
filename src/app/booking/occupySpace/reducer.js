import { OCCUPY_SPACE, OCCUPY_SPACE_SUCCESS, OCCUPY_SPACE_FALURE } from './actions';

const initialState = {
  occupyResponse: {},
};

// eslint-disable-next-line default-param-last
function occupyReducer(state = initialState, action) {
  // eslint-disable-next-line no-param-reassign
  state.occupyResponse = {};
  switch (action.type) {
    case OCCUPY_SPACE:
      return {
        ...state,
        occupyResponse: (state.occupyResponse, { loading: true }),
      };
    case OCCUPY_SPACE_SUCCESS:
      return {
        ...state,
        occupyResponse: (state.occupyResponse,
        { loading: false, data: action.payload.data, err: {} }),
      };
    case OCCUPY_SPACE_FALURE:
      return {
        ...state,
        occupyResponse: (state.occupyResponse,
        { loading: false, err: action.error.data, data: {} }),
      };
    default:
      return state;
  }
}

export default occupyReducer;
