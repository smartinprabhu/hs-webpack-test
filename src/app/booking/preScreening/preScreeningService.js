import { questionaireDataGetCall, preScreeningPostCall } from './actions';

export function getQuestionaireData(checkListId) {
  return (dispatch) => {
    dispatch(questionaireDataGetCall(checkListId));
  };
}

export function saveProcessedPreScreening(data) {
  return (dispatch) => {
    dispatch(preScreeningPostCall(data));
  };
}
