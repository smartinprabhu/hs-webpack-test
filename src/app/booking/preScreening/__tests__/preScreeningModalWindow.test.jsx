import React from 'react';
import { useSelector } from 'react-redux';
import { fireEvent, render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import PreScreensModalWindow from '../preScreeningModalWindow';
import * as actions from '../actions';
import * as services from '../preScreeningService';
import api from '../../../../middleware/api';
import reducer from '../reducer';
import 'babel-polyfill';

jest.mock('react-redux', () => ({
  useSelector: jest.fn((fn) => fn()),
  useDispatch: () => jest.fn(),
  useState: jest.fn(),
}));

const userInfo = {
  vendor: {
    id: 1450,
    name: 'Motivity Labs(Tenant)',
  },
  employee: {
    id: '123',
  },
  company: {
    address: 'Water Mark  Building\n222 W. Las Colinas Blvd.\nSuite 755 East\nIrving TX 75039\nUnited States',
    id: 3,
    name: 'Water Mark  Building',
    timezone: 'US/Central',
  },
};
const bookingData = {
  access_status: false,
  actual_in: '',
  actual_out: '',
  book_for: 'myself',
  booking_type: 'individual',
  company: {
    id: 63,
    name: 'Water Mark Building',
  },
  employee: {
    id: 6453,
    name: 'SI ABHI',
  },
  id: 91404,
  is_host: false,
  members: [],
  planned_in: '2020-10-20 01:45:00',
  planned_out: '2020-10-20 10:15:00',
  planned_status: 'Regular',
  prescreen_status: true,
  shift: {
    id: 47,
    name: 'A',
  },
  space: {
    id: 9428,
    latitude: '432.75',
    longitude: '132',
    name: 'WS#41',
    status: 'Ready',
  },
  user_defined: false,
  uuid: 'a517dca6-3910-46fd-aed0-b0bfc48426b9',
  vendor: {
    id: 1450,
    name: 'Motivity Labs(Tenant)',
  },
  working_hours: 0,
};

describe('PreScreensModalWindow testing', () => {
  const openModalWindow = jest.fn();
  it('snapshot testing while having data', () => {
    useSelector.mockImplementation((fn) => fn({
      preScreening: {
        questionarieData: {
          data: [{
            id: 16811,
            mro_quest_grp_id: {},
            name: 'Do you have Cough?',
            type: 'boolean',
          }],
          loading: false,
          err: {},
        },
        preScreeningProcess: {
          data: [2558, 2559, 2560],
          loading: false,
          err: {},
        },
      },
      user: {
        userInfo,
      },
      config: {
        configObj: {
          data: {
            prescreen: [{ id: 1711, name: 'Covid-19 Checklist.' }],
          },
        },
      },
    }));
    const { asFragment } = render(
      <PreScreensModalWindow
        openModalWindow={openModalWindow}
        modalWindowOpen
        bookingData={bookingData}
      />,
    );
    expect(asFragment()).toMatchSnapshot();
  });
  it('snapshot testing while having questionarieData error', () => {
    useSelector.mockImplementation((fn) => fn({
      preScreening: {
        questionarieData: {
          data: [{
            id: 16811,
            mro_quest_grp_id: {},
            name: 'Do you have Cough?',
            type: 'boolean',
          }],
          loading: false,
          err: {},
        },
        preScreeningProcess: {
          data: {},
          loading: false,
          err: 'error',
        },
      },
      user: {
        userInfo,
      },
      config: {
        configObj: {
          data: {
            prescreen: [{ id: 1711, name: 'Covid-19 Checklist.' }],
          },
        },
      },
    }));
    const { asFragment } = render(
      <PreScreensModalWindow
        openModalWindow={openModalWindow}
        modalWindowOpen
        bookingData={bookingData}
      />,
    );
    expect(asFragment()).toMatchSnapshot();
  });
  it('snapshot testing while having preScreeningProcess error', () => {
    useSelector.mockImplementation((fn) => fn({
      preScreening: {
        questionarieData: {
          data: {},
          loading: false,
          err: 'error',
        },
        preScreeningProcess: {
          data: [2558, 2559, 2560],
          loading: false,
          err: {},
        },
      },
      user: {
        userInfo,
      },
      config: {
        configObj: {
          data: {
            prescreen: [{ id: 1711, name: 'Covid-19 Checklist.' }],
          },
        },
      },
    }));
    const { asFragment } = render(
      <PreScreensModalWindow
        openModalWindow={openModalWindow}
        modalWindowOpen
        bookingData={bookingData}
      />,
    );
    expect(asFragment()).toMatchSnapshot();
  });
  const create = () => {
    const store = jest.fn();
    const next = jest.fn();
    const invoke = (action) => api(store)(next)(action);
    return { next, invoke };
  };

  it('passes through GET_QUESTIONARY_DATA_SUCCESS action', () => {
    const { next, invoke } = create();
    const action = { type: actions.GET_QUESTIONARY_DATA_SUCCESS };
    invoke(action);
    expect(next).toHaveBeenCalledWith(action);
  });

  it('passes through GET_QUESTIONARY_DATA_ERROR action', () => {
    const { next, invoke } = create();
    const action = { type: actions.GET_QUESTIONARY_DATA_ERROR };
    invoke(action);
    expect(next).toHaveBeenCalledWith(action);
  });

  it('passes through PRE_SCREENING_PROCESS_SUCCESS action', () => {
    const { next, invoke } = create();
    const action = { type: actions.PRE_SCREENING_PROCESS_SUCCESS };
    invoke(action);
    expect(next).toHaveBeenCalledWith(action);
  });

  it('passes through PRE_SCREENING_PROCESS_ERROR action', () => {
    const { next, invoke } = create();
    const action = { type: actions.PRE_SCREENING_PROCESS_ERROR };
    invoke(action);
    expect(next).toHaveBeenCalledWith(action);
  });

  it('it should handle questionaireDataGetCall', () => {
    const questionaireDataGetCall = () => actions.questionaireDataGetCall();
    questionaireDataGetCall();
    expect(actions.questionaireDataGetCall).toBeDefined();
  });

  it('it should handle preScreeningPostCall', () => {
    const preScreeningPostCall = () => actions.preScreeningPostCall();
    preScreeningPostCall();
    expect(actions.preScreeningPostCall).toBeDefined();
  });

  it('it should handle getQuestionaireData', () => {
    const getQuestionaireData = () => services.getQuestionaireData();
    getQuestionaireData();
    expect(services.getQuestionaireData).toBeDefined();
  });

  it('it should handle saveProcessedPreScreening', () => {
    const saveProcessedPreScreening = () => services.saveProcessedPreScreening();
    saveProcessedPreScreening();
    expect(services.saveProcessedPreScreening).toBeDefined();
  });

  it('it should handle saveProcessedPreScreening function', async () => {
    const dispatch = jest.fn();
    const spy = jest.spyOn(actions, 'preScreeningPostCall');
    const result = services.getQuestionaireData();
    spy.mockImplementation(() => Promise.resolve());
    await result(dispatch);
    expect(dispatch.mock.calls[0]).toBeDefined();
  });

  it('it should handle saveProcessedPreScreening function', async () => {
    const dispatch = jest.fn();
    const spy = jest.spyOn(actions, 'preScreeningPostCall');
    const result = services.saveProcessedPreScreening();
    spy.mockImplementation(() => Promise.resolve());
    await result(dispatch);
    expect(dispatch.mock.calls[0]).toBeDefined();
  });

  const initialState = {
    questionarieData: {},
    preScreeningProcess: {},
  };

  it('it should handle PRE_SCREENING_PROCESS state', () => {
    expect(reducer(initialState, { type: actions.PRE_SCREENING_PROCESS })).toEqual({
      questionarieData: {},
      preScreeningProcess: { loading: true },
    });
  });

  it('it should handle PRE_SCREENING_PROCESS_SUCCESS state', () => {
    expect(reducer(initialState, {
      type: actions.PRE_SCREENING_PROCESS_SUCCESS,
      payload: { data: [2558, 2559, 2560] },

    })).toEqual({
      questionarieData: {},
      preScreeningProcess: {
        loading: false, data: [2558, 2559, 2560], err: {},
      },
    });
  });

  it('it should handle PRE_SCREENING_PROCESS_ERROR state', () => {
    expect(reducer(initialState, {
      type: actions.PRE_SCREENING_PROCESS_ERROR,
      error: { data: 'error' },
    })).toEqual({
      questionarieData: {},
      preScreeningProcess: { loading: false, err: 'error', data: {} },
    });
  });

  it('it should handle GET_QUESTIONARY_DATA state', () => {
    expect(reducer(initialState, { type: actions.GET_QUESTIONARY_DATA })).toEqual({
      questionarieData: { loading: true },
      preScreeningProcess: {},
    });
  });

  it('it should handle GET_QUESTIONARY_DATA_SUCCESS state', () => {
    expect(reducer(initialState, {
      type: actions.GET_QUESTIONARY_DATA_SUCCESS,
      payload: {
        data: [{
          id: 16811,
          mro_quest_grp_id: {},
          name: 'Do you have Cough?',
          type: 'boolean',
        }],
      },

    })).toEqual({
      preScreeningProcess: {},
      questionarieData: {
        loading: false,
        data: [{
          id: 16811,
          mro_quest_grp_id: {},
          name: 'Do you have Cough?',
          type: 'boolean',
        }],
        err: {},
      },
    });
  });

  it('it should handle GET_QUESTIONARY_DATA_ERROR state', () => {
    expect(reducer(initialState, {
      type: actions.GET_QUESTIONARY_DATA_ERROR,
      payload: 'error',
    })).toEqual({
      preScreeningProcess: {},
      questionarieData: { loading: false, err: 'error', data: {} },
    });
  });

  it('dom testing while initiating the prescreening', () => {
    useSelector.mockImplementation((fn) => fn({
      preScreening: {
        questionarieData: {
          data: [{
            id: 16811,
            constr_mandatory: true,
            name: 'Do you have Cough?',
            type: 'boolean',
          }, {
            id: 16810,
            constr_mandatory: true,
            name: 'Do you have fever?',
            type: 'numerical_box',
          }, {
            id: 16809,
            constr_mandatory: true,
            name: 'Do you have cold?',
            type: 'textbox',
          }],
          loading: false,
          err: {},
        },
        preScreeningProcess: {
          loading: true,
        },
      },
      user: {
        userInfo,
      },
      config: {
        configObj: {
          data: {
            prescreen: { checklists: [{ id: 1711, name: 'Covid-19 Checklist.' }] },
          },
        },
      },
    }));
    const { container, getByText } = render(
      <PreScreensModalWindow
        openModalWindow={openModalWindow}
        modalWindowOpen
        bookingData={bookingData}
      />,
    );
    fireEvent.click(getByText('Next'));
    fireEvent.click(getByText('Back'));
    expect(container).toBeInTheDocument();
  });

  it('dom testing while having preScreeningProcess error', () => {
    useSelector.mockImplementation((fn) => fn({
      preScreening: {
        questionarieData: {
          data: [{
            id: 16811,
            constr_mandatory: true,
            name: 'Do you have Cough?',
            type: 'boolean',
          }],
          loading: false,
          err: {},
        },
        preScreeningProcess: {
          data: [],
          loading: false,
          err: {
            error: {
              message: 'error',
            },
          },
        },
      },
      user: {
        userInfo,
      },
      config: {
        configObj: {
          data: {
            prescreen: { checklists: [{ id: 1711, name: 'Covid-19 Checklist.' }] },
          },
        },
      },
    }));
    const { container, getByText } = render(
      <PreScreensModalWindow
        openModalWindow={openModalWindow}
        modalWindowOpen
        bookingData={bookingData}
      />,
    );
    expect(getByText('error')).toBeInTheDocument();
    expect(container).toBeInTheDocument();
  });

  it('dom testing while performing complete screening', () => {
    useSelector.mockImplementation((fn) => fn({
      preScreening: {
        questionarieData: {
          data: [
            {
              id: 16810,
              constr_mandatory: true,
              name: 'Do you have fever?',
              type: 'numerical_box',
            }, {
              id: 16809,
              constr_mandatory: true,
              name: 'Do you have cold?',
              type: 'textbox',
            }],
          loading: true,
          err: {},
        },
        preScreeningProcess: {
          loading: true,
        },
      },
      user: {
        userInfo,
      },
      config: {
        configObj: {
          data: {
            prescreen: { checklists: [{ id: 1711, name: 'Covid-19 Checklist.' }] },
          },
        },
      },
    }));
    const { container, getByText, getByTestId } = render(
      <PreScreensModalWindow
        openModalWindow={openModalWindow}
        modalWindowOpen
        bookingData={bookingData}
      />,
    );
    fireEvent.click(getByText('Next'));
    fireEvent.change(getByTestId('16809'), { target: { value: 'test data' } });
    fireEvent.change(getByTestId('16810'), { target: { value: 123 } });
    fireEvent.click(getByText('Next'));
    fireEvent.click(getByText('Next'));
    fireEvent.click(getByText('Finish'));
    expect(container).toBeInTheDocument();
  });

  it('dom testing while performing complete screening', () => {
    useSelector.mockImplementation((fn) => fn({
      preScreening: {
        questionarieData: {
          data: [{
            id: 16811,
            constr_mandatory: true,
            name: 'Do you have Cough?',
            type: 'boolean',
          }, {
            id: 16810,
            constr_mandatory: true,
            name: 'Do you have fever?',
            type: 'numerical_box',
          }, {
            id: 16809,
            constr_mandatory: true,
            name: 'Do you have cold?',
            type: 'textbox',
          }],
          loading: true,
          err: {},
        },
        preScreeningProcess: {
          loading: true,
        },
      },
      user: {
        userInfo,
      },
      config: {
        configObj: {
          data: {
            prescreen: { checklists: [{ id: 1711, name: 'Covid-19 Checklist.' }] },
          },
        },
      },
    }));
    const { container, getByText, getByTestId } = render(
      <PreScreensModalWindow
        openModalWindow={openModalWindow}
        modalWindowOpen
        bookingData={bookingData}
      />,
    );
    fireEvent.click(getByText('Next'));
    fireEvent.click(getByText('Next'));
    fireEvent.click(getByText('Yes'));
    fireEvent.click(getByText('No'));
    fireEvent.change(getByTestId('16809'), { target: { value: 'test data' } });
    fireEvent.click(getByText('Next'));
    expect(container).toBeInTheDocument();
  });
});
