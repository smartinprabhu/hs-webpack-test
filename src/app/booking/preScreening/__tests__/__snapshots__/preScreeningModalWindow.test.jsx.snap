// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PreScreensModalWindow testing snapshot testing while having data 1`] = `<DocumentFragment />`;

exports[`PreScreensModalWindow testing snapshot testing while having preScreeningProcess error 1`] = `<DocumentFragment />`;

exports[`PreScreensModalWindow testing snapshot testing while having questionarieData error 1`] = `<DocumentFragment />`;
